faker/providers/address/uk_UA/__init__.py

```diff
@@ -651,6 +650,7 @@ class Provider(AddressProvider):
         "Південний Судан",
         "Польща",
         "Португалія",
+        "Респу́бліка Півні́чна Македо́нія",
```

Comment:
This is actually cursed. Symbols, like those on letters `у́`, `ч` (lol, GitHub doesn't recognize it because it's an illegal symbol) and `о́` are not wrote in words. Those are used only for marking pronunciation in Ukrainian. So the right name here should be:

```
Північна Республіка Македонія
```

P.S. I see that some translations lack word `Republic`, how about add `TODO` comment to all of the changed lines, with content like `Please review this string, should be "Republic of North Macedonia". See #1809`.
P.S.2 I see that some languages have more than 2 changes, what does it mean?

Reply:
Thank you noticing @PerchunPak, I'll fix this entry.